spec/models/project_spec.rb

```diff
@@ -504,4 +504,29 @@
       end
     end
   end
+
+  describe "#communication_emails" do
+    let(:project) { create(:project) }
+    let(:owner_email) { project.owner.email }
+
+    it 'should return the owner by default' do
+      expect(project.communication_emails).to match_array([owner_email])
+    end
+
+    context "with communication project roles" do
+      let(:comms_user) { create(:user) }
+      before do
+        create(
+          :access_control_list,
+          user_group: comms_user.identity_group,
+          resource: project,
+          roles: ["communications"]
+        )
+      end
+
+      it 'should return the owner by default' do
```

Comment:
By "default", do you mean it should just be first in the array? Or is the description here wrong?